src/UKHO.ERPFacade.API/Controllers/ErpFacadeController.cs

```diff
@@ -7,20 +10,48 @@ namespace UKHO.ERPFacade.API.Controllers
     public class ErpFacadeController : BaseController<ErpFacadeController>
     {
         private readonly ILogger<ErpFacadeController> _logger;
+        private readonly IAzureTableReaderWriter _azureTableReaderWriter;
+        private readonly IAzureBlobEventWriter _azureBlobEventWriter;
+
+        public const string TRACEIDKEY = "data.traceId";
+        private const string UPDATE_RESPONSE_TIME = "ResponseDateTime";
+
 
         public ErpFacadeController(IHttpContextAccessor contextAccessor,
-                                   ILogger<ErpFacadeController> logger)
+                                   ILogger<ErpFacadeController> logger,
+                                   IAzureTableReaderWriter azureTableReaderWriter,
+                                   IAzureBlobEventWriter azureBlobEventWriter)
         : base(contextAccessor)
         {
             _logger = logger;
+            _azureTableReaderWriter = azureTableReaderWriter;
+            _azureBlobEventWriter = azureBlobEventWriter;
         }
 
         [HttpPost]
-        [Route("/erpfacade/getpriceinfo")]
-        public virtual async Task<IActionResult> Post()
+        [Route("/erpfacade/priceinformation")]
```

Comment:
Please add [Authorize] attribute on call and [Authorize(Policy = "PriceInformationApiCaller")] on endpoint.

Reply:
this change will be done in upcoming sprints.